ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -1,12 +1,26 @@
 import ivy
 
 
-def cholesky(x, /, *, symmetrize_input=True):
-    def symmetrize(x):
-        # TODO : Take Hermitian transpose after complex numbers added
-        return (x + ivy.swapaxes(x, -1, -2)) / 2
+def _symmetrize(x):
+    # TODO : Take Hermitian transpose after complex numbers added
```

Comment:
Hi @atlasrule. Thanks for your contribution. Is this TODO label needed for further usage?

Reply:
Hi @MrZilinXiao, the TODO label needed for further usage, since complex dtypes not currently supported. Thank you!

For reference:
https://github.com/unifyai/ivy/pull/4370#issuecomment-1245120642
https://lets-unify.ai/ivy/functional/ivy/linear_algebra/svd/svd_functional.html?highlight=complex